docs/docs/overview/quickstart/get-started-with-nestjs.md

```diff
@@ -0,0 +1,286 @@
+---
+sidebar_position: 10
+sidebar_label: Get started with NestJS
+---
+
+# NestJS Quickstart
+
+Learn how to use Novu to quickly send multi-channel (SMS, Email, Chat, Push) notifications from a NestJS app.
+
+In this Quickstart, you will learn how to:
+
+- Install the Novu Node.js SDK via npm.
+- Use Novu Node.js SDK with NestJS dependency injection.
+- Send your first notification.
```

Comment:
The following sections are missing:

- Topics
- Create A Topic
- Add Subscribers to a Topic
- Send Notification to a Topic

Reply:
I had to wait some time for npm release of novu node sdk with my fix https://github.com/novuhq/novu/pull/3795

Missing sections were added in recent commits